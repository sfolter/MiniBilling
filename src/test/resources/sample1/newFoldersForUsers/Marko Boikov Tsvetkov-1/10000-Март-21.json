{
  "documentDate": "2022-07-27T15:15:11Z",
  "reference": "1",
  "totalAmount": 156.59999,
  "documentNumber": 10000,
  "lines": [
    {
      "amount": 156.59999,
      "product": "gas",
      "quantity": 87.0,
      "lineStart": "2021-03-17T13:20:00+03:00",
      "price": 1.8,
      "index": 1,
      "lineEnd": "2021-03-17T13:20:00+03:00",
      "priceList": 1
    }
  ],
  "consumer": "Marko Boikov Tsvetkov"
}