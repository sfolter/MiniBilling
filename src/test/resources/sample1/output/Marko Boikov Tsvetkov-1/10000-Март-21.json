{
    "documentDate": "2022-08-18T17:45:04.007616",
    "documentNumber": "10000",
    "consumer": "Marko Boikov Tsvetkov",
    "reference": "1",
    "totalAmount": "80.07 EUR",
    "totalAmountWithVat": "169.72 EUR",
    "lines": [{
        "index": 1,
        "quantity": 87,
        "lineStart": "2021-01-01T09:00:00Z",
        "lineEnd": "2021-03-17T10:20:00Z",
        "product": "gas",
        "price": 1.8,
        "priceList": 1,
        "amount": "80.07 EUR"
    }],
    "taxes": [{
        "index": 1,
        "lines": 1,
        "name": "Standing charge",
        "quantity": 75,
        "unit": "days",
        "price": 1.6,
        "amount": "61.36 EUR"
    }],
    "vat": [
        {
            "index": 1,
            "lines": 1,
            "taxes": 0,
            "taxedAmountPercentage": 60,
            "percentage": 20,
            "amount": "9.61 EUR"
        },
        {
            "index": 1,
            "lines": 1,
            "taxes": 0,
            "taxedAmountPercentage": 40,
            "percentage": 20,
            "amount": "6.41 EUR"
        },
        {
            "index": 1,
            "lines": 0,
            "taxes": 1,
            "taxedAmountPercentage": 100,
            "percentage": 20,
            "amount": "12.27 EUR"
        }
    ]
}