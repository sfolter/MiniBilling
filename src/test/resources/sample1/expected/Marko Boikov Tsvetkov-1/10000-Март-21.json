{
  "documentNumber": "10000",
  "consumer": "Marko Boikov Tsvetkov",
  "reference": "1",
  "totalAmount": 156.6,
  "lines": [
    {
      "index": 1,
      "quantity":87,
      "lineStart": "2021-01-01T09:00:00Z",
      "lineEnd": "2021-03-17T10:20:00Z",
      "product": "gas",
      "price": 1.8,
      "priceList": 1,
      "amount": 156.6
    }
  ]
}
